### ISSUE TITLE
Add support for force_3_0_protocol_version

### ISSUE THREAD

**Issue by rehevkor5** [michaelklishin/cassandra-chef-cookbook#378]
This issue keeps track of the need to add support for the "cassandra.force_3_0_protocol_version" property explained here https://docs.datastax.com/en/upgrade/doc/upgrade/datastax_enterprise/upgdDseMinor50x.html for updating from DSE 5.0.0-5.0.8 to 5.0.9 and later 5.0.x releases.

It looks like this also applies to Apache Cassandra, too: https://issues.apache.org/jira/browse/CASSANDRA-13004

Is templates/default/cassandra-env.sh.erb JVM_OPTS  the right place for this?

**Comment by michaelklishin** [comment]
@rehevkor5 yes, we configure things such as `cassandra.jmx.local.port` there so I'd say it belongs there. Free free to submit a PR. Thank you!
